docs/sql_reference/src/asciidoc/_chapters/olap_functions.adoc

```diff
@@ -26,8 +26,20 @@
 [[olap_functions]]
 = OLAP Functions
 
-This section describes the syntax and semantics of the On Line
-Analytical Process (OLAP) window functions. The OLAP window functions
+On-line Analytical Process (OLAP) functions provide the capability to partition source data into groups of rows, order the rows in each group, compute aggregates within a window frame for a row. Typically, an OLAP function in SQL is expressed in two components <FUNCTION> and <OVER>: <FUNCTION> specifies the function and the arguments, and <OVER> specifies the partitioning, the ordering and the window frames. <OVER> is sometimes called the window. OLAP functions are used to perform analytic tasks against data sets stored in or computed from a set of tables, such as moving average and ranks.
+
+Trafodion also supports a set of sequence functions which are SQL extensions and have to be used together with a SEQUENCE BY clause. These sequence functions can be used to compute a running quantity of a column up to and include the current row as follows.

```

Comment:
because we have Chapter 7 to describe 'sequence functions', maybe here no need to list all the functions , such as RUNNINGAVG, and only put a reference link ?